WISE-Community/WISE-Client
# feat(Authoring): Add lesson buttons navigate to add lesson route

## Changes
- Clicking on the add lesson buttons next to each of the lessons navigates to the add lesson route
- Deleted the "Add New Lesson" button at the top of the Authoring Tool
- Deleted choose lesson location route components
- Fixed a bug when adding lesson structure to a unit that has no steps

## Test
- Make sure adding a lesson works
- Make sure the target location is persisted even if you go forward and back between different structure types
  - For example
    - Click the add lesson button
    - Click Jigsaw
    - Click Back
    - Click Self-Directed Investigation
    - Click Submit
    - This should work
- Go to or create a unit that has no steps
  - Add a lesson (this used to throw an error but now it should work properly)

Closes #1720
